src/i18n/_locales/ja/messages.json

```diff
@@ -146,9 +277,169 @@
   "i18n_display_format" : {
     "message" : "表示形式"
   },
+  "i18n_spread_auto" : {
+     "message" : "自動"
+  },
+  "i18n_scroll_mode" : {
+    "message" : "スクロールモード"
+  },
+  "i18n_scroll_mode_auto" : {
+    "message" : "自動"
+  },
+  "i18n_scroll_mode_doc" : {
+    "message" : "Document"
+  },
+  "i18n_scroll_mode_continuous" : {
+    "message" : "Continuous"
+  },
+
+  "i18n_page_transition" : {
+    "message" : "ページ効果"
+  },
+  "i18n_page_transition_none" : {
+    "message" : "無効"
+  },
+  "i18n_page_transition_fade" : {
+    "message" : "フェード"
+  },
+  "i18n_page_transition_slide" : {
+    "message" : "スライド"
+  },
+  "i18n_page_transition_swoosh" : {
+    "message" : "スウォッシュ"
+  },
+  "i18n_page_transition_butterfly" : {
+    "message" : "バタフライ"
+  },
   "i18n_html_readium_tm_a_project" : {
     "message" : "Readium<sup>TM</sup>とは, International Digital Publishing Forum (IDPF) と支援企業によるプロジェクトで, EPUB&reg; 出版のためのシステムとレンダリングエンジンのオープンソースリファレンスです。詳しくは、<a href=\"http://readium.org/\">プロジェクトホームページ</a>をご覧ください。これまでは<a href=\"http://evidentpoint.com/\">Evident Point</a>と<a href=\"http://www.bluefirereader.com/\">Bluefire Productions</a>がプロジェクト開発をリードしてきました。 コントリビュートするには<a href=\"https://github.com/readium/readium-js-viewer\">githubレポジトリ</a>をご覧ください。"
   },
+"i18n_alt_about_logos" : {
+    "message" : "HTML5, i d p f, ePUB"
+  },
+  "i18n_toolbar" : {
+    "message" : "ツールバー"
+  },
+  "i18n_toolbar_show" : {
+    "message" : "ツールバーを表示"
+  },
+  "i18n_toolbar_hide" : {
+    "message" : "ツールバーを隠す"
+  },
+  "i18n_audio_play" : {
+    "message" : "再生"
+  },
+  "i18n_audio_pause" : {
+    "message" : "停止"
+  },
+  "i18n_audio_play_background" : {
+    "message" : "バックグラウンドで音声再生"
+  },
+  "i18n_audio_pause_background" : {
+    "message" : "バックグラウンドの音声停止"
+  },
+  "i18n_audio_previous" : {
+    "message" : "前の音声フレーズ"
+  },
+  "i18n_audio_next" : {
+    "message" : "次の音声フレーズ"
+  },
+  "i18n_audio_volume" : {
+    "message" : "音量"
+  },
+  "i18n_audio_volume_increase" : {
+    "message" : "音を大きく"
+  },
+  "i18n_audio_volume_decrease" : {
+    "message" : "音を小さく"
+  },
+  "i18n_audio_time" : {
+    "message" : "Audio time cursor"
```

Comment:
This is a message that could also be translated for "audio timeline". Is that easier to translate in Japanese?